### PR TITLE
Add support for the Ropsten beacon chain

### PR DESCRIPTION
**Pull request by zah**

### CODE REVIEW

File: beacon_chain/networking/network_metadata.nim [hunk 1]
```diff
@@ -217,11 +228,16 @@ template eth2Network(path: string, eth1Network: Eth1Network): Eth2NetworkMetadat
...
+template mergeTestnet(path: string, eth1Network: Eth1Network): Eth2NetworkMetadata =
+  loadCompileTimeNetworkMetadata(mergeTestnetsDir & "/" & path,
+                                 some eth1Network)
+
 when not defined(gnosisChainBinary):
   when const_preset == "mainnet":
     const
       mainnetMetadata* = eth2Network("shared/mainnet", mainnet)
       praterMetadata* = eth2Network("shared/prater", goerli)
+      ropstenMetadata = mergeTestnet("ropsten-beacon-chain", ropsten)
```
Comment (arnetheduck) [inline beacon_chain/networking/network_metadata.nim:240]: just `ropsten`?

Comment (arnetheduck) [reply beacon_chain/networking/network_metadata.nim:240]: ah, nvm, they renamed the config but not the folder

**Review by arnetheduck — APPROVED** [review]
cc @unixpi this will need a corresponding update in the book where we show users how to set up nimbus with an EL post-merge

File: beacon_chain/networking/network_metadata.nim (hunk 1 above)
Comment (zah) [reply beacon_chain/networking/network_metadata.nim:240]: This is a name of a metadata directory in a third-party repo.

**Pull request merged**